rs/nns/sns-wasm/src/sns_wasm.rs

```diff
@@ -3629,7 +3631,7 @@ mod test {
                 (root_id, vec![this_id.get(), governance_id.get()]),
```

Comment:
This should be in its own section, not be mixed in with root-controlled canisters.

Reply:
I looked into it, unfortunately the calls appear in a fixed order, and I'd have to respecify the tests to compare sets rather than vectors. I don't think this is worth doing in this PR.